azure/src/main/java/org/apache/iceberg/azure/AzureProperties.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.azure;
+
+import java.io.Serializable;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
+import org.apache.iceberg.relocated.com.google.common.collect.Maps;
+
+public class AzureProperties implements Serializable {
+
+  // Start of storage account configuration
+  public static final String STORAGE_CONNECTION_STRING = "azure.storage.%s.connection-string";
```

Comment:
These seem long. What about `abfs.%s.uri` or `abfs.%s.connection-string` instead? Does it really need to be "connection string"?

Reply:
> These seem long. What about abfs.%s.uri or abfs.%s.connection-string instead?

I can replace `azure.storage` with `abfs` for this and the rest of the configs to make them shorter.

> Does it really need to be "connection string"?

It would be a good idea to keep `connection-string` as is, since it would directly map to the `connection-string` config available on the Azure portal, thereby reducing the chances of confusing this config with a different one.